src/pdf/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -118,13 +120,20 @@ const LabelCard: React.FC<LabelCardProps> = ({ data, index, quantity }) => {
                     <View style={[styles.leftCol, styles.bottomAlign]}>
                         <Text style={styles.largeText}>{data.address_postcode}</Text>
                     </View>
-                    <View style={[styles.middleCol, styles.bottomAlign]}>
+                    <View style={[styles.middleCol, styles.bottomAlign, { flexDirection: "row" }]}>
                         <Text style={styles.mediumText}>
                             {data.packing_slot} |
                             {data.collection_centre !== "DLVR"
-                                ? data.collection_centre // TODO VFB-56 needs icon
-                                : "Delivery"}
+                                ? data.collection_centre + " "
+                                : "Delivery "}
                         </Text>
+                        <FontAwesomeIconPdfComponent
+                            faIcon={
+                                data.collection_centre !== "DLVR"
+                                    ? faBuildingCircleArrowRight
+                                    : faTruck
```

Comment:
the logic is probs clearer this way?
```suggestion
                                data.collection_centre === "DLVR"
                                    ? faTruck
                                    : faBuildingCircleArrowRight
```